### ISSUE TITLE
Datepicker reading out unnecessary or addtional text for screenreader users when hit enter to select date

### ISSUE THREAD

**Issue by PavanJagini** [microsoft/fluentui#9821]
## Actual behavior:
When screen reader users navigate "From" and "To" Datepicker controls and select any date from the calendar then the name of the "My Action History" tab is announced along with the "From" and "To" controls which is inappropriate.
![image](https://user-images.githubusercontent.com/52062804/61289259-907df000-a7e6-11e9-9ba5-03ae6fecc909.png)

## Expected behavior:
On selecting any of the date from the calendar for "From" and "To" controls the extra unnecessary information should not be announced on the "From" and "To" control and the selected date should be announced for screen reader users.

**Comment by joschect** [comment]
@PavanJagini thanks for your post. Could you provide a codepen that reproduces this behavior? Here is a basic [datepicker codepen](https://codepen.io/slimemold/pen/wLVxjK) to get you started.

**Comment by joschect** [comment]
Also, are you passing "My Action History" as a label to the datepicker in any way? If it's not being passed through then it is more likely that something is going on with narrator rather than there being a problem with the date picker itself.

**Comment by PavanJagini** [comment]
Hey @joschect, Thanks for responding. Below is the exact code snippet used. please have a look and suggest. Thanks in adv.

                    <div className="col-md-2 col-xs-6 col-sm-4">
                        <DatePicker
                            label="From"
                            placeholder="yyyy-mm-dd"
                            isRequired={false}
                            allowTextInput={true}
                           // ariaLabel={desc}
                            firstDayOfWeek={firstDayOfWeek}
                            strings={UUMCONSTANTS.MY_ACTION_HISTORY_CALENDAR}
                            value={fromValue ? fromValue : undefined}
                            onSelectDate={this._onFromSelectDate}
                            formatDate={this._onFormatDate}
                             
                        />
                    </div>
                    <div className="col-md-2 col-xs-6 col-sm-4">
                        <DatePicker
                            label="To"
                            placeholder="yyyy-mm-dd"
                            isRequired={false}
                            allowTextInput={true}
                            //ariaLabel={desc}
                            firstDayOfWeek={firstDayOfWeek}
                            strings={UUMCONSTANTS.MY_ACTION_HISTORY_CALENDAR}
                            value={toValue ? toValue : undefined}
                            onSelectDate={this._onToSelectDate}
                            formatDate={this._onFormatDate}
                          
                        />
                    </div>

**Comment by joschect** [comment]
Hey @PavanJagini it doesn't look like you are passing in a string to datepicker into one of the aria props. I do see that you had description but it's comment out so it's unlikely that the datepicker is the reason that the extra text is being read out. Are you able to make a repro using the codepen starter? It would help out a lot if you could go there and the structure that you're using for the "My Action History" title so we can narrow down the source of the problem.

**Comment by PavanJagini** [comment]
Hi @joschect, I spent some time on building scenario on code pen but it is about loading the grid dynamically with data finding hard to do so. As I am also internal to Microsoft, would be possible to have a call to give a demo of implementation? please confirm. Thanks.

**Comment by joschect** [comment]
@PavanJagini I was able to reproduce it in this codepen, https://codepen.io/slimemold/pen/wLVxjK . Can you confirm that you're using fabric pivots as well?

**Comment by PavanJagini** [comment]
Hey @joschect , THanks a lot for repro the issue for me! Yes, we are using Pivots for attaining tabs view as shown in the actual behaviour section. PFB the screenshot( for readability purpose) & code snippet of the code for Pivots implementation. Please suggest.

                        <Pivot className="active-roles-tabs" linkFormat={PivotLinkFormat.links} linkSize={PivotLinkSize.normal}>
                            {shouldRenderActionHistory ? <PivotItem linkText="My Pending Actions" ariaLabel="My Pending Actions">
                                <MyActions />
                            </PivotItem>
                                :
                                <div></div>
                            }
                            {shouldRenderActionHistory ?
                                <PivotItem linkText="My Action History" ariaLabel="My Action History">
                                    <AuditTrail />
                                </PivotItem>
                                : <div></div>
                            }
                            <PivotItem linkText="My Request History" ariaLabel="My Request History">
                                <MyRequestHistory />
                            </PivotItem>
                        </Pivot>

![image](https://user-images.githubusercontent.com/52062804/61849833-2fce7180-aed0-11e9-92ee-ca2896cf7a85.png)

**Comment by joschect** [comment]
@PavanJagini this is definitely a bug that we'll need to look into. Currently I'm not quite sure why it's happening, my best guess is that focus is momentarily going to the pivot header when the calendar dismisses.

**Comment by PavanJagini** [comment]
Thanks for confirming @joschect, I shall take this info to accessibility team to get the bug closed.

**Comment by joschect** [comment]
@PavanJagini This is an issue with fabric, I think, we still need to investigate though.
